src/test/java/org/ccci/obiee/client/rowmap/RowmapIntegrationTest.java

```diff
@@ -169,12 +168,58 @@ public void testSortByDate() throws Exception
         SaiDonationRow previous = null;
         for (SaiDonationRow row : rows) {
             if (previous != null)
-                assertThat(row.getTransactionDate(), greaterThanOrEqualTo(previous.getTransactionDate()));
+                assertThat(row.getTransactionDate(), isOnOrAfter(previous.getTransactionDate()));
             previous = row;
         }
         printRowsize(rows);
     }
-    
+
+    private Matcher<LocalDate> isOnOrAfter(final LocalDate date)
+    {
+        return new TypeSafeDiagnosingMatcher<LocalDate>()
+        {
+            public void describeTo(Description description)
+            {
+                description.appendText("is on or after").appendValue(date);
+            }
+
+            @Override
+            protected boolean matchesSafely(LocalDate item, Description mismatchDescription)
+            {
+                boolean onOrAfter = !item.isBefore(date);
+                if (!onOrAfter)
+                {
+                    mismatchDescription.appendValue(item);
+                }
+                return onOrAfter;
+            }
+        };
+    }
+
+    private Matcher<LocalDate> isOnOrBefore(final LocalDate date)
+    {
+        return new TypeSafeDiagnosingMatcher<LocalDate>()
+        {
+            public void describeTo(Description description)
+            {
+                description.appendText("is before or on").appendValue(date);
+            }
+
+            @Override
+            protected boolean matchesSafely(LocalDate item, Description mismatchDescription)
+            {
+                boolean beforeOrOn = !item.isAfter(date);
```

Comment:
I think it sounds more natural the other way. If this was standalone I wouldn't think much of it, but since the other variable name exists, I think they should be consistent (one way or the other).